src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -62,13 +132,38 @@ function LHNOptionsList({style, contentContainerStyles, data, onSelectRow, optio
      *
      * @return {Component}
      */
-    const renderItem = ({item}) => (
-        <OptionRowLHNDataWithFocus
-            reportID={item}
-            viewMode={optionMode}
-            shouldDisableFocusOptions={shouldDisableFocusOptions}
-            onSelectRow={onSelectRow}
-        />
+    const renderItem = useCallback(
+        ({item: reportID}) => {
+            const itemFullReport = reports[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`] || {};
+            const itemReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`];
+            const itemParentReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport.parentReportID}`];
+            const itemPolicy = policy[`${ONYXKEYS.COLLECTION.POLICY}${itemFullReport.policyID}`];
+            const itemTransaction = `${ONYXKEYS.COLLECTION.TRANSACTION}${lodashGet(
+                itemParentReportActions,
+                [itemFullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'],
+                '',
+            )}`;
+            const itemComment = draftComments[`${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${reportID}`] || '';
+            const participantPersonalDetailList = _.values(OptionsListUtils.getPersonalDetailsForAccountIDs(itemFullReport.participantAccountIDs, personalDetails));
+            return (
+                <OptionRowLHNData
+                    reportID={reportID}
+                    fullReport={itemFullReport}
+                    reportActions={itemReportActions}
+                    parentReportActions={itemParentReportActions}
+                    policy={itemPolicy}
+                    personalDetails={participantPersonalDetailList}
+                    transaction={itemTransaction}
+                    receiptTransactions={transactions}
+                    viewMode={optionMode}
+                    isFocused={!shouldDisableFocusOptions && reportID === currentReportID}
+                    onSelectRow={onSelectRow}
+                    preferredLocale={preferredLocale}
```

Comment:
After adding `preferredLocale` to `renderItem` we should have added it to the FlatList's `extraData` too otherwise the list won't re-render

Reply:
Probably a false positive, the above is true for `FlashList` only. For `FlatList` the change in `renderItem` itself seems enough